src/Microsoft.OData.Client/BulkUpdateSaveResult.cs

```diff
@@ -28,6 +31,31 @@ internal class BulkUpdateSaveResult : BaseSaveResult
         /// </summary>
         private readonly BulkUpdateGraph bulkUpdateGraph;
 
+        /// <summary> 
+        /// Link descriptors for descriptors that were added to the entity tracker during an $expand operation.

```

Comment:
"Link descriptors for descriptors" - what does that mean? Or do you "Link descriptors associated with entity descriptors"?

Reply:
yes this ~ Link descriptors associated with entity descriptors

Reply:
Maybe you should have a shorter summary and the longer explanation in a `<remark>`